src/game/overlay.c

```diff
@@ -80,21 +80,14 @@ static RGBA8888 m_ColorBarMap[][COLOR_STEPS] = {
       { 210, 130, 160, 255 },
       { 165, 100, 120, 255 },
       { 120, 60, 70, 255 } },
+    // purple
+    { { 52, 22, 80, 255 },
+      { 70, 30, 107, 255 },
+      { 52, 22, 80, 255 },
+      { 39, 17, 60, 255 },
+      { 26, 11, 40, 255 } },
```

Comment:
Just making a note that in the future we might as well add support for another bar color in the config tool.

Btw, to determine the purple colors, I used the PS1 menu color as the brightest highlight and then used [this site](https://www.w3schools.com/colors/colors_picker.asp?colorhex=461e6b) to get darker percentages.